File: QB_Challenges/PrepInsta/01_Getting_Started/04_Sum_Between_Range/Java/SORN_Formulae.java
import java.util.Scanner;

public class SORN_Formulae {
    
    public static void sorn() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the first number: ");
        int num1 = sc.nextInt();
        System.out.print("Enter the second number: ");
        int num2 = sc.nextInt();
        sc.close();

        int sum = num2*(num2+1)/2 - num1*(num1+1)/2 + num1;
        System.out.println("The sum of "+num1+" & "+num2+" is: "+sum);
    }

    public static void main(String[] args) {
        sorn();
    }
}

File: QB_Challenges/PrepInsta/01_Getting_Started/04_Sum_Between_Range/CPP/SORN_Formulae.cpp
#include <iostream>
using namespace std;

void SORN(){
    int num1, num2;
    cout <<"Enter the first number: ";
    cin >> num1;
    cout <<"Enter the second number: ";
    cin >> num2;

    int sum = num2*(num2+1)/2 - num1*(num1+1)/2 + num1;

    cout <<"The sum of "<<num1<<" & "<<num2<<" is: "<<sum<<endl;
}

int main(){
    SORN();
    return 0;
}